Servlets/src/test/java/ua/goit/servlets/servlets/TestProjectsServlet.java

```diff
@@ -0,0 +1,44 @@
+package ua.goit.servlets.servlets;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.io.FileUtils;
+import org.mockito.Mockito;
+
+import ua.goit.servlets.servlets.Projects;
+
+public class TestProjectsServlet extends Mockito {
+	 Projects servlet;
+
+	@Before
+	public void setUp() throws ServletException {
+		servlet = new Projects();
+		servlet.init();
+	}
+
+	@Test
+	public void testServlet() throws Exception {
+		HttpServletRequest request = mock(HttpServletRequest.class);       
+		HttpServletResponse response = mock(HttpServletResponse.class); 
+		when(request.getRequestURI()).thenReturn("http://localhost:8080/Servlets/projects?category=ART");
+		when(request.getParameter("category")).thenReturn("ART");
+		PrintWriter writer = new PrintWriter("fileServletProjects.txt");
```

Comment:
Please try StringWriter here, you won't need the temp file - tests will become faster too